src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -66,13 +66,13 @@ internal sealed class TextBufferIssueTracker : IIssueTracker, IIssueConsumer
         public TextBufferIssueTracker(DTE dte, TaggerProvider provider, ITextDocument document,
             IEnumerable<AnalysisLanguage> detectedLanguages, IIssuesFilter issuesFilter,
             ISonarErrorListDataSource sonarErrorDataSource, ILogger logger)
-            : this(dte, provider, document, detectedLanguages, issuesFilter, sonarErrorDataSource, logger, new IssueSpanCalculator())
+            : this(dte, provider, document, detectedLanguages, issuesFilter, sonarErrorDataSource, logger, new IssueToIssueMarkerConverter())
         {
         }
 
         internal TextBufferIssueTracker(DTE dte, TaggerProvider provider, ITextDocument document,
             IEnumerable<AnalysisLanguage> detectedLanguages, IIssuesFilter issuesFilter,
-            ISonarErrorListDataSource sonarErrorDataSource, ILogger logger, IIssueSpanCalculator issueSpanCalculator)
+            ISonarErrorListDataSource sonarErrorDataSource, ILogger logger, IIssueToIssueMarkerConverter issueMarkerConverter)
```

Comment:
The span calculator is used internally by the new type, so the tracker doesn't need to know about it.